packages/sitecore-jss-nextjs/src/components/NextImage.tsx

```diff
@@ -85,7 +85,7 @@ export const NextImage: React.FC<NextImageProps> = ({
   const loader = (otherProps.loader ? otherProps.loader : sitecoreLoader) as ImageLoader;
 
   if (attrs) {
-    return <Image loader={loader} {...imageProps} />;
+    return <Image alt="" loader={loader} {...imageProps} />;
```

Comment:
Any test case needed for `alt` default of empty string?

Reply:
Good catch! Done